pymic/offload_stream.py

```diff
@@ -562,6 +562,9 @@ def invoke(self, kernel, *args):
             raise OffloadError("Cannot invoke kernel, "
                                "library not loaded on device")
 
+        # Map None to nullptr
+        args = [arg if arg is not None else 0 for arg in args]
```

Comment:
So my understanding is that mapping to zero results in a pointer to zero being passed to the kernel.  While scalars are passed by pointer I think that this is the best we can do.